sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -0,0 +1,444 @@
+   * console.log(deletedRoleAssignment);
+   * ```
+   * @summary Deletes an existing role assignment.
+   * @param roleScope The scope of the role assignment.
+   * @param name The name of the role assignment.
+   * @param [options] The optional parameters.
+   */
+  public async deleteRoleAssignment(
+    roleScope: RoleAssignmentScope,
+    name: string,
+    options?: DeleteRoleAssignmentOptions
+  ): Promise<KeyVaultRoleAssignment> {
+    const requestOptions = operationOptionsToRequestOptionsBase(options || {});
+    const span = createSpan("deleteRoleAssignment", requestOptions);
+
+    let response: RoleAssignmentsDeleteResponse;
+    try {
+      response = await this.client.roleAssignments.delete(
+        this.vaultUrl,
+        roleScope,
+        name,
+        setParentSpan(span, requestOptions)
+      );
+    } finally {
+      span.end();
+    }
+
+    return mappings.roleAssignment.generatedToPublic(response);
+  }
+
+  /**
+   * Gets a role assignments previously created in an Azure Key Vault.
+   *
+   * Example usage:
+   * ```ts
+   * const client = new KeyVaultAccessControlClient(url, credentials);
+   * let roleAssignment = await client.createRoleAssignment("/", "295c179b-9ad3-4117-99cd-b1aa66cf4517");
+   * roleAssignment = const await client.getRoleAssignment(roleAssignment.properties.roleScope, roleAssignment.name);
+   * console.log(roleAssignment);
+   * ```
+   * @summary Gets an existing role assignment.
+   * @param roleScope The scope of the role assignment.
+   * @param name The name of the role assignment.
+   * @param [options] The optional parameters.
+   */
+  public async getRoleAssignment(
+    roleScope: RoleAssignmentScope,
+    name: string,
+    options?: GetRoleAssignmentOptions
+  ): Promise<KeyVaultRoleAssignment> {
+    const requestOptions = operationOptionsToRequestOptionsBase(options || {});
+    const span = createSpan("getRoleAssignment", requestOptions);
+
+    let response: RoleAssignmentsDeleteResponse;
+    try {
+      response = await this.client.roleAssignments.get(
+        this.vaultUrl,
+        roleScope,
+        name,
+        setParentSpan(span, requestOptions)
+      );
+    } finally {
+      span.end();
+    }
+
+    return mappings.roleAssignment.generatedToPublic(response);
+  }
+
+  /**
+   * @internal
+   * @ignore
+   * Deals with the pagination of {@link listRoleAssignments}.
+   * @param roleScope The scope of the role assignments.
+   * @param continuationState An object that indicates the position of the paginated request.
+   * @param [options] Common options for the iterative endpoints.
+   */
+  private async *listRoleAssignmentsPage(
+    roleScope: RoleAssignmentScope,
+    continuationState: ListRoleAssignmentsPageSettings,
+    options?: ListRoleAssignmentsOptions
+  ): AsyncIterableIterator<KeyVaultRoleAssignment[]> {
+    if (continuationState.continuationToken == null) {
```

Comment:
can't this be a truthy check?

```suggestion
    if (continuationState.continuationToken) {
```

Or is the idea that continuationToken could be a falsy value like 0 or ""  and still be valid?

Reply:
This comes from code on other clients. I can take your feedback nonetheless.

Reply:
It would be with the `!` at the beginning though 😛  right?

Reply:
er yes